src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int OPERATION_TYPE_POSITION = 0;
+    public static final int AMOUNT_POSITION = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String example = "";
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+            StringBuilder dateWithFile = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                dateWithFile.append(value).append(System.lineSeparator());
+                value = bufferedReader.readLine();
+            }
+            example = getExample(dateWithFile.toString());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file" + fromFileName, e);
+        }
+        getWriterWithFile(example, toFileName);
```

Comment:
this method should look like this:
`List<String> dataFromFile = readFromFile(fromFileName);`
`String report = getReport(dataFromFile);`
`writeToFile(report, toFileName);`

method readFromFile() should only read data from file
getReport() should create report, parse everything? calculate... and return ready report as string
writeToFile() should oly get some string and write it to file
those methods should be private